### DISCUSSION TITLE
DDT4ALL does not find my OBD2 device (USB) With Renault Grand Scenic 2017

### DISCUSSION THREAD

**Discussion by Dannermax** [cedricp/ddt4all#955] [Q&A]
Hello all.

I am trying to connect my Renault Grand Scenic (2017) through an OBD2 device (with the 25k80 FTDI chip) to my paltop using DDT4ALL.

My computer finds the usb device, but no USB devices shows up in the DDT4ALL app. The OBD2 device only has a red light, lighting constantly.

I have tried the different communications ports in DDT4ALL but nothing. 

I also have a chinese clone OBD2 device (WIFI) does not show up either in the DDT4ALL software. But i can connect to it from my iphone OBD2 app... and it works fine. there. But i need to use the DDT4ALL software in order to activate apple car play.

I bought this UBS OBD2 device:

[https://lifafadenmark.com/search?q=OBD2+Diagnostic+Scanner+ELM327+V1.4+USB+ELM+327+med+PIC18F25K80+Chip&options%5Bprefix%5D=last](url)

![OBD2](https://github.com/cedricp/ddt4all/assets/3067087/1ae0d42c-cd7c-488e-b850-4fc4ba6bc078)

Whan can i do? Really apprechiate any help i can get here.. Thanks :-)

**Comment by KarelSvo** [comment]
For a USB interface, a suitable driver must be installed. This will assign a USB port. Check the interface in Windows Device Manager.
The interface must be paired before starting ddt4all so that it can be selected.
I recommend [lifafadenmark.com](https://lifafadenmark.com/products/vgate-vlinker-fs-for-forscan-adapter-hsms-can-obd2-diagnostic-scanner?_pos=2&_sid=a14daac67&_ss=r)

For the WiFi interface you must to click on the WiFi symbol to connect to the WiFi address.

**Comment by Dannermax** [comment]
Hi.

Really apprechiate the help. When i first plugged the OBD device to my laptop it told me it was installed correctly.

But if i plug it into my laptop now, i cant seem to find it anywhere. Do you know where i have to look? 
![obd2-1](https://github.com/cedricp/ddt4all/assets/3067087/212a2d40-16ab-41f8-9627-423a6b2bfbcb)

**Comment by Dannermax** [comment]
On the cd the device came with there are drivers for the usb device. Which one do you think is the correct for my device?

On the back of the dvice it said: 25k80 FTDI on a sticker.
![obd3](https://github.com/cedricp/ddt4all/assets/3067087/aabe935b-ae8b-496b-bdd0-573a454ace84)

**Reply by AndyIreland** [reply]
right click on the ones in device manager with a exclamation mark, choose 'update driver' - point to the CD (or folder where the drivers are) choose include sub folders and it should find the drivers automatically and install them
